dev/com.ibm.ws.request.interrupt/src/com/ibm/ws/request/interrupt/internal/InterruptibleThreadInfrastructureImpl.java

```diff
@@ -395,4 +392,15 @@ public void introspect(PrintWriter out) throws Exception {
 			}
 		}
 	}
+
+	/**
+	 * Used for testing/debugging purposes.
+	 * @return size of threadMap
+	 */
+	public int getThreadsMapSize() {
+		synchronized (threadMap) {
+			return threadMap.size();
+		}
```

Comment:
```suggestion
		return threadMap.size();
```
The map is already a synchronized map so do not need to add synchronized around it.